### DISCUSSION TITLE
Add web push notifications to Grafana Alerting

### DISCUSSION THREAD

**Discussion by ghost** [grafana/grafana#67978] [Alerting]
Please add desktop notifications in Grafana. This will make it a much better tool for using as monitoring our servers and application. If this will be added we can use it in our production environment.

**Comment by bergquist** [comment]
Not sure what you mean by desktop notifications. Something like http://growl.info/ ?

That would require a client application running the desktop that connects to Grafana.

**Comment by jimmyd-be** [comment]
I think the user is saying to use desktop notifications like Chrome and Firefox does for some websites. A js library that can show the notifications can be https://github.com/ttsvetko/HTML5-Desktop-Notifications.

This is also a feature that I want to use Grafana as our monitor tool for our servers/applications.

**Comment by andreyradygin** [comment]
This would be really nice feature!

**Comment by james-mchugh** [comment]
I am surprised that this issue has not gotten more attention. This feature would be extremely useful

**Comment by haldi4803** [comment]
I was wondering when i couldn't find the "Web Push" option in the Notifications.
I mean.... why not use the Webserver that i do have running anyway, to send notifications to the browser i log in from wherever. And with Chrome it even works when the Browser is closed!


@torkelo uhm yeah... the Title "Desktop Notifications" is somewhat faulty, it should be "Web Push Notifications", as you only need your browser as client, no extra desktop software. Works even on mobilphones.
https://developers.google.com/web/fundamentals/push-notifications/how-push-works
https://stackoverflow.com/questions/33687298/how-to-send-push-notification-to-web-browser

**Comment by sscots** [comment]
+1

**Comment by armandgrillet** [comment]
This is an interesting feature request for Grafana Alerting as it could be a very user-friendly way to set up alerts. We have not worked on this yet and I will move it to a GitHub discussion for now.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82941. Feel free to continue the discussion around this there. Thank you!
